site/components/Header/HeaderMobile.tsx

```diff
@@ -35,6 +35,17 @@ export const HeaderMobile: FC<Props> = (props) => {
           onClick={() => setIsToggled(!isToggled)}
         />
       </header>
+      <Global
+        styles={
+          isToggled &&
+          css`
+            body {
+              position: fixed;
+              height: 100vh;
+            }
+          `
+        }
+      />
```

Comment:
All we need is `overflow: hidden`, which also solves the issue of losing your scroll position.

But the downside with that is it hides the scrollbar, which causes a tiny layout shift. No way around it.

lastly, nitpick, I would just move the conditional around the entire tag so we don't render it unless its being used:

```suggestion
    {isToggled && (
      <Global
        styles={css`
            body {
              overflow-y: hidden;
            }
          `
        }
      />)}
```

Reply:
Nice, overflow-y works much better.
We could customize the scroll bar for site the same way we did for app and give the body a padding-right of the scroll bar width to avoid the layout shift.